pyfstat/core.py

```diff
@@ -1565,6 +1565,28 @@ def __init__(
         self.init_computefstatistic()
         self.init_semicoherent_parameters()
 
+    def _init_semicoherent_window_trick(self):
+        """
+        Use this window to compute the semicoherent Fstat using TransientFstatMaps.
+        This way we are able to decouple this clever shenanigan from the actual
```

Comment:
While I agree this is a "clever shenanigan", I don't think that's good documentation style. ;)